sway-lib-std/src/token.sw

```diff
@@ -165,7 +178,7 @@ pub fn burn(amount: u64) {
 /// transfer(500, BASE_ASSET_ID, to_address);
 /// transfer(500, BASE_ASSET_ID, to_contract_id);
 /// ```
-pub fn transfer(amount: u64, asset_id: AssetId, to: Identity) {
```

Comment:
Instead of removing `AssetId`, should we just make a separate struct that's distinct from `ContractId`?

And if we do want to get rid of `AssetId`, we should probably remove it from `contract_id.sw` as well.

Reply:
I was going to keep `AssetId` and update the constant type